api/Element.json

```diff
@@ -4870,9 +4860,7 @@
             },
             "safari_ios": "mirror",
             "samsunginternet_android": "mirror",
-            "webview_android": {
-              "version_added": "≤37"
-            },
+            "webview_android": "mirror",
```

Comment:
Another case where mirroring yields partial support, but this is probably right.